src/ResultHistory.re

```diff
@@ -39,6 +72,15 @@ let make = (~communityName: string) => {
     setDateTo(dt => dt->Belt.Option.map(d => d->DateFns.addWeeks(-1)));
   };
 
+  let queryData =
+    Query.use(~variables={communityName, dateFrom, dateTo}, ());
+
+  let resultsFragment = queryData.results_connection.fragmentRefs;
+
+  let communitySettingsFragment =
+    queryData.community_settings_connection.edges->Belt.Array.getExn(0).node.
```

Comment:
Is `getExn` going to throw for newly created communities?

Reply:
Fixed in https://github.com/Yakimych/relogify/pull/103/commits/fcb61cb5a6d464d5a1efa2632f6a02af64bfe501